### PR TITLE
Jenkins

### PR DESCRIPTION
**Pull request by jonhattan**

### CODE REVIEW

File: manifests/jenkins.pp [hunk 1]
```diff
@@ -1,40 +1,62 @@
...
+    exec { 'ducktape-jenkins-bootstrap-cmd':
+      command => "${puppet_helper} set_security full_control \
+        && ${puppet_helper} create_or_update_user '${admin_user}' '${admin_mail}' '${admin_pass}' '${admin_name}' '${admin_keys}' \
+        && ${remove_admin} \
+        && rm ${initial_admin_pass_file}",
       notify => Class['Jenkins::Cli::Reload'],
     }
+
+    #TODO# Generate token for admin user.
+    file {$::jenkins::cli_password_file:
```
Comment (NITEMAN) [inline manifests/jenkins.pp:54]: @jonhattan `$::jenkins::cli_password_file` [defaults to undef on upstream module](https://github.com/voxpupuli/puppet-jenkins/blob/master/manifests/init.pp#L319) , we probably should do some sanity check here

Comment (NITEMAN) [reply manifests/jenkins.pp:54]: Or maybe this should point to [`$jenkins::cli::config::cli_password_file`](https://github.com/voxpupuli/puppet-jenkins/blob/master/manifests/cli/config.pp#L18)

Comment (jonhattan) [reply manifests/jenkins.pp:54]: Moved to automagic.

**Pull request merged**